wire-ios-data-model/Source/ManagedObjectContext/Migration/2.106-2.107/DuplicateClientsMigrationPolicy.swift

```diff
@@ -44,6 +48,7 @@ class DuplicateClientsMigrationPolicy: NSEntityMigrationPolicy {
 
             clients.first?.setValue(true, forKey: Keys.needsToBeUpdatedFromBackend.rawValue)
             clients.dropFirst().forEach(context.delete)
+            zmLog.safePublic("removed 1 occurence of duplicate clients", level: .warn)
```

Comment:
well, removed clients.length -1 occurrences

Reply:
I have just copied the comment.. but think it is meant to be called repeatedly in the for loop and just say "multiple duplicated clients occurred 1 time for this identifier" and then there can be more for other identifiers.